_pages/tutorial.markdown

```diff
@@ -16,37 +16,193 @@ toc: true
 
 # Tutorial
 
-System TermIt is designed to manage terms and vocabularies, assign them to documents and other resources and interconnect them. Basic tutorial focuses on those activities, describing step by step how to:
-* search for terms and browse vocabularies,
-* look for occurrences and definitions in documents,
-* create new vocabulary and set of terms,
-* assigning document to the vocabulary and annotate it.
+System TermIt is designed for terms and vocabularies management. It allows users to assign terms from the specific vocabularies to the documents or any other sources and interconnect them. The basic tutorial describes step by step following activities:
```

Comment:
TermIt is designed ...

Reply:
... or any other resources ...

Reply:
The basic tutorial describes the following activities step by step:

Reply:
done, step by step is deleted